nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -2197,9 +2150,8 @@ define([
                         doEditNotebookMetadata();
                     }));
 
-                    busEventManager.add(bus.on('actionButton', function(message) {
-                        doActionButton(message.data);
-                    }));
```

Comment:
I'm not sure where, but somewhere in the action button initiation, it sets the button in Run mode to be active in the app cell (not bulk import) as soon as it starts. So it looks like it's ready to run, when it's not.

Reply:
Thanks for finding that. I will take a look